src/Controls/src/Core/Platform/Windows/Styles/Resources.xaml

```diff
@@ -4,7 +4,6 @@
 	x:Class="Microsoft.Maui.Controls.Platform.Resources">
 
     <ResourceDictionary.MergedDictionaries>
-        <ResourceDictionary Source="MauiControlsCommandBarStyle.xaml" />
```

Comment:
AFAICT this style is never actually used. The CommandBar is never initialized with the "MauiControlsCommandBarStyle" style and the styles in this file are all explicit styles